Documentation/reports.md

```diff
@@ -242,9 +242,9 @@ spec:
 ```
 
 ### runImmediately
+When `runImmediately` is set to `true`, the report will be run immediately. This is a simple way to have a report get processed and queued right away without setting schedule values that could cause it to get requeued.
 
-Set `runImmediately` to `true` to run the report immediately with all available data, regardless of the `reportingStart` or `reportingEnd` values, and without checking if there is any data for the report period.
-For reports with a schedule set, it will not wait for each period's reportingEnd to elapse before processing and all reportPeriods between `reportingStart` and `reportingEnd`.
+> *Note*: You must set a `reportingEnd` value when `runImmediately` is set to `true`. If you do not specify a `reportingStart' value, the current time will be used.
```

Comment:
@bentito it's good to have the note, but FYI, if the user leaves that value of, the validation will flag it 

> pruan@MacBook-Pro ~/workspace/v3-testfiles/metering (master●)$ oc create -f run_now_example.yaml
> The Report "namespace-memory-usage-now" is invalid:
> * : Invalid value: "": "spec" must validate at least one schema (anyOf)
> * spec.reportingEnd: Required value